Documentation/installation/bandwidth-manager.rst

```diff
@@ -170,6 +170,14 @@ the ``netperf-server`` Pod):
 Each Pod is represented in Cilium as an :ref:`endpoint` which has an identity. The above
 identity can then be correlated with the ``cilium endpoint list`` command.
 
+.. note::
+
+   Bandwidth limits apply on a per-pod scope. Each pod in the identity gets its own
+   bandwidth allocation.
```

Comment:
This is confusing as there is only one Cilium Endpoint Identity (such as 491 above) per Cilium-managed pod.